text/0000-javascript-module-api.md

```diff
@@ -0,0 +1,1131 @@
+| `import GlobalsResolver from "@ember/application/globals-resolver"` | `Ember.DefaultResolver` |
+| `import Resolver from "@ember/application/resolver"`                | `Ember.Resolver`        |
+
+### `@ember/array`
+| Module                                            | Global               |
+| ---                                               | ---                  |
+| `import EmberArray from "@ember/array"`           | `Ember.Array`        |
+| `import { A } from "@ember/array"`                | `Ember.A`            |
+| `import { isArray } from "@ember/array"`          | `Ember.isArray`      |
+| `import { makeArray } from "@ember/array"`        | `Ember.makeArray`    |
+| `import MutableArray from "@ember/array/mutable"` | `Ember.MutableArray` |
+| `import ArrayProxy from "@ember/array/proxy"`     | `Ember.ArrayProxy`   |
+
+### `@ember/component`
+| Module                                                | Global                |
+| ---                                                   | ---                   |
+| `import Component from "@ember/component"`            | `Ember.Component`     |
+| `import Checkbox from "@ember/component/checkbox"`    | `Ember.Checkbox`      |
+| `import Helper from "@ember/component/helper"`        | `Ember.Helper`        |
+| `import { helper } from "@ember/component/helper"`    | `Ember.Helper.helper` |
+| `import TextArea from "@ember/component/text-area"`   | `Ember.TextArea`      |
+| `import TextField from "@ember/component/text-field"` | `Ember.TextField`     |
+
+### `@ember/controller`
+| Module                                       | Global                    |
+| ---                                          | ---                       |
+| `import Controller from "@ember/controller"` | `Ember.Controller`        |
+| `import { inject } from "@ember/controller"` | `Ember.inject.controller` |
+
+### `@ember/debug`
+| Module                                                                     | Global                        |
+| ---                                                                        | ---                           |
+| `import { assert } from "@ember/debug"`                                    | `Ember.assert`                |
+| `import { debug } from "@ember/debug"`                                     | `Ember.debug`                 |
+| `import { inspect } from "@ember/debug"`                                   | `Ember.inspect`               |
+| `import { runInDebug } from "@ember/debug"`                                | `Ember.runInDebug`            |
+| `import { warn } from "@ember/debug"`                                      | `Ember.warn`                  |
+| `import ContainerDebugAdapter from "@ember/debug/container-debug-adapter"` | `Ember.ContainerDebugAdapter` |
+| `import DataAdapter from "@ember/debug/data-adapter"`                      | `Ember.DataAdapter`           |
```

Comment:
Should `onerror` be importable from this package? I don't see it in the RFC anywhere.

https://guides.emberjs.com/v2.9.0/configuring-ember/debugging/#toc_implement-an-ember-onerror-hook-to-log-all-errors-in-production